From 969454eb0aa7515880bcbc7a16e458570ae7fb82 Mon Sep 17 00:00:00 2001
From: Ecco park <eccopark@broadcom.com>
Date: Thu, 14 Aug 2014 17:25:15 -0700
Subject: [PATCH 60/77] net: wireless : bcmdhd fix the type of bssidx of P2P

Currently we set WL_INVALID(-1) to bssidx for deinit.
However, the bssdix has u32 type.
so, we need to change type to s32.
Signed-off-by: Ecco park <eccopark@broadcom.com>

Change-Id: Icf18dbe6cbed560f07ecb2c25d5e8261021fabee
---
 drivers/net/wireless/bcmdhd/wl_cfgp2p.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/bcmdhd/wl_cfgp2p.h b/drivers/net/wireless/bcmdhd/wl_cfgp2p.h
index fc6b85fbc224..f4c7c4ff9b14 100644
--- a/drivers/net/wireless/bcmdhd/wl_cfgp2p.h
+++ b/drivers/net/wireless/bcmdhd/wl_cfgp2p.h
@@ -74,7 +74,7 @@ struct p2p_saved_ie {
 };
 
 struct p2p_bss {
-	u32 bssidx;
+	s32 bssidx;
 	struct net_device *dev;
 	struct p2p_saved_ie saved_ie;
 	void *private_data;
-- 
2.0.3

